_data/de.yml

```
@@ -53,7 +53,7 @@ homepage_content:
     <li class="mt-0 mb-4 pb-4 border-bottom step step-text-2" data-image="step-image-2">Fügen Sie Ihre Konten für Landes- oder Kryptowährungen hinzu.</li> <li class="mt-0 mb-4 pb-4 border-bottom step step-text-3" data-image="step-image-3">Erstellen Sie ein neues Angebot oder nehmen Sie ein bestehendes an.</li> <li class="mt-0 step step-text-4" data-image="step-image-4">Folgen Sie der Benutzeroberfläche, um den Handel abzuschließen.</li>
   hJoin: Treten Sie der Community bei
   pJoin: >
-    <p>Bisq ist immer auf der Suche nach hochqualifizierten Entwicklern, Designern und Sicherheitsexperten, die unserer <a href="https://bisq.network/de/community/">Community</a>beitreten möchten. Wenn Sie daran interessiert sind, Bisq besser zu machen, gibt es viele Möglichkeiten, <a href="https://docs.bisq.network/contributor-checklist.html" target="_blank" rel="noopener">einen Beitrag zu leisten</a>. Wir würden uns freuen,  <a href="https://bisq.network/de/community/">von Ihnen zu hören</a>!</p> <p>Unsere <a href="https://bisq.network/roadmap/">Roadmap</a> gibt einen Überblick über High-Level-Funktionen und Meilensteine auf dem Weg zu Bisq v1.0.</p>
+    <p>Bisq ist immer auf der Suche nach hochqualifizierten Entwicklern, Designern und Sicherheitsexperten, die unserer <a href="https://bisq.network/de/community/">Community</a>beitreten möchten. Wenn Sie daran interessiert sind, Bisq besser zu machen, gibt es viele Möglichkeiten, <a href="https://docs.bisq.network/contributor-checklist.html" target="_blank" rel="noopener">einen Beitrag zu leisten</a>. Wir würden uns freuen,  von Ihnen zu hören!</p> <p>Unsere <a href="https://bisq.network/roadmap/">Roadmap</a> gibt einen Überblick über High-Level-Funktionen und Meilensteine auf dem Weg zu Bisq v1.0.</p>
```

```suggestion
    <p>Bisq ist immer auf der Suche nach hochqualifizierten Entwicklern, Designern und Sicherheitsexperten, die unserer <a href="https://bisq.network/de/community/">Community</a>beitreten möchten. Wenn Sie daran interessiert sind, Bisq besser zu machen, gibt es viele Möglichkeiten, <a href="https://docs.bisq.network/contributor-checklist.html" target="_blank" rel="noopener">einen Beitrag zu leisten</a>. Wir würden uns freuen, von Ihnen zu hören!</p> <p>Unsere <a href="https://bisq.network/roadmap/">Roadmap</a> gibt einen Überblick über High-Level-Funktionen und Meilensteine auf dem Weg zu Bisq v1.0.</p>
```

Really minor nit, but there's an extra space here.